src/video/wayland/SDL_waylandevents_c.h

```diff
@@ -29,6 +29,36 @@
 #include "SDL_waylanddatamanager.h"
 #include "SDL_waylandkeyboard.h"
 
+#ifdef SDL_VIDEO_DRIVER_WAYLAND_TABLET
+struct SDL_WaylandTabletSeat;
```

Comment:
Is there a known type that we can use in place of SDL_WaylandTabletSeat?

Reply:
there is `zwp_tablet_seat_v2` which could be used. But then I'd need to include "tablet-unstable-v2-protocol.h". I think I didn't want to do this, because I assumed you didn't want to include third party headers in SDL (like is done with the QT in `SDL_waylandvideo.h`.

Reply:
Would it be possible to do "struct zwp_tablet_seat_v2" so it doesn't hit the undeclared error? I suppose we could adjust the ordering so protocol files are included earlier... trying to remember how I did this for the other types.

Reply:
No, because I need to cast it back to `zwp_tablet_seat_v2` in the `SDL_waylandevents.c` and I don't seem to be able to include "tablet-unstable-v2-protocol.h" in there:
(basically a bunch of)
```
In file included from /home/admin/workspace/games/sdl/SDL/src/video/wayland/SDL_wayland[...].c:26:
/home/admin/workspace/games/sdl/SDL/src/video/wayland/SDL_waylandevents_c.h:33:10: fatal error: tablet-unstable-v2-protocol.h: No such file or directory
   33 | #include "tablet-unstable-v2-protocol.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```